### ISSUE TITLE
Azure terraform output - Parse error

### ISSUE THREAD

**Issue by virtualbubble** [chris-rock/inspec-verify-provision#1]
I tried to test inspec with Terraform using the output JSON but get parse errors. See below. Is this supported with Azure resources?

> inspec exec .\inspec-azure\ -t azure://
```
[2018-03-17T18:27:02+00:00] WARN: Overwriting resource azure_generic_resource. To reference a specific version of azure_generic_resource use the resource() method
[2018-03-17T18:27:02+00:00] WARN: Overwriting resource azure_resource_group. To reference a specific version of azure_resource_group use the resource() method
[2018-03-17T18:27:02+00:00] WARN: Overwriting resource azure_virtual_machine. To reference a specific version of azure_virtual_machine use the resource() method
[2018-03-17T18:27:02+00:00] WARN: Overwriting resource azure_virtual_machine_data_disk. To reference a specific version of azure_virtual_machine_data_disk use the resource() method
Traceback (most recent call last):
        20: from C:/tools/ruby25/bin/inspec:23:in `<main>'
        19: from C:/tools/ruby25/bin/inspec:23:in `load'
        18: from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/inspec-2.0.45/bin/inspec:12:in `<top (required)>'
        17: from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/base.rb:466:in `start'
        16: from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor.rb:387:in `dispatch'
        15: from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/invocation.rb:126:in `invoke_command'
        14: from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/thor-0.20.0/lib/thor/command.rb:27:in `run'
        13: from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/inspec-2.0.45/lib/inspec/cli.rb:168:in `exec'
        12: from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/inspec-2.0.45/lib/inspec/runner.rb:103:in `run'
        11: from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/inspec-2.0.45/lib/inspec/runner.rb:82:in `load'
        10: from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/inspec-2.0.45/lib/inspec/runner.rb:82:in `each'
         9: from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/inspec-2.0.45/lib/inspec/runner.rb:93:in `block in load'
         8: from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/inspec-2.0.45/lib/inspec/profile.rb:164:in `collect_tests'
         7: from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/inspec-2.0.45/lib/inspec/profile.rb:164:in `each'
         6: from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/inspec-2.0.45/lib/inspec/profile.rb:167:in `block in collect_tests'
         5: from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/inspec-2.0.45/lib/inspec/profile_context.rb:141:in `load_control_file'
         4: from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/inspec-2.0.45/lib/inspec/profile_context.rb:157:in `load_with_context'
         3: from C:/tools/ruby25/lib/ruby/gems/2.5.0/gems/inspec-2.0.45/lib/inspec/profile_context.rb:157:in `instance_eval'
         2: from ./inspec-azure/controls/example.rb:8:in `load_with_context'
         1: from C:/tools/ruby25/lib/ruby/2.5.0/json/common.rb:156:in `parse'
C:/tools/ruby25/lib/ruby/2.5.0/json/common.rb:156:in `parse': 765: unexpected token at '��{' (JSON::ParserError)
```

**My controls.rb is as follows:**

```
encoding: utf-8
copyright: 2018, The Authors

load data from Terraform output
content = inspec.profile.file("terraform.json")
params = JSON.parse(content)

control 'azure_vm_example' do
    title 'Ensure the example VM matches expectations'
    impact 1.0
  
    describe azure_generic_resource(group_name: 'example-group-name', name: 'test') do
      # Ensure the VM is located in the correct region
      its('location') { should cmp 'eastus2' }
  
      # Ensure the VM has tags
      it { should have_tags }
    end
end
```

**The terraform.json is as follows:**

```json
{
    "vnet_id1": {
        "sensitive": false,
        "type": "list",
        "value": [
            "/subscriptions/<subscription_number>/resourceGroups/<resource_group_name>/providers/Microsoft.Network/virtualNetworks/VNET-TEST"
        ]
    }
}
```

**Comment by chris-rock** [comment]
@virtualbubble Thank you for the question. I had no problem parsing the array:

```
# load data from terraform output
content = inspec.profile.file("terraform.json")
params = JSON.parse(content)

ARRAY = params['vnet_id1']['value']
```
Be aware that your terraform.json is not a valid json file. You need curly brackets:

```
{
    "vnet_id1": {
        "sensitive": false,
        "type": "list",
        "value": [
            "/subscriptions/<subscription_number>/resourceGroups/<resource_group_name>/providers/Microsoft.Network/virtualNetworks/VNET-TEST"
        ]
    }
}
```
Let me know if that worked.

**Comment by virtualbubble** [comment]
I decided to look back into this issue and found that the problem was when using Visual Studio Code and running Terraform in the console built into the IDE to run the output to JSON it was outputting it using different encoding. Running Terraform output from a command prompt it worked. 

I found this when opening the files to compare a working version of the JSON nd a broken version. The contents were identical but the encoding was different. 

![terra_encode](https://user-images.githubusercontent.com/35614911/39409992-4a534e8c-4be8-11e8-8fdf-31a418906204.PNG)
